utils/utils.go

```diff
@@ -310,3 +310,13 @@ func GetVariantVersion(version, imageVariant string) string {
 	}
 	return fmt.Sprintf("%s-%s", version, imageVariant)
 }
+
+// Returns image version and variant.
+// Expected imageTag format: <version>-<variant>, i.e. 1.0.0-mariner.
+func GetVersionAndImageVariant(imageTag string) (string, string) {
```

Comment:
What if the version is 1.9.0-rc.2-mariner?

Reply:
yes, Thanks for catching. I think we should use `LastIndex` here also, right ?

Reply:
Yes, that should work.